backend/src/main/java/com/woowacourse/levellog/member/application/MemberService.java

```diff
@@ -23,14 +24,24 @@ public class MemberService {
 
     @Transactional
     public Long save(final MemberCreateDto request) {
+        checkSameGithubId(request);
+
         final Member member = new Member(request.getNickname(), request.getGithubId(), request.getProfileUrl());
```

Comment:
```suggestion
        final Member member = request.toMember();
```
서비스에서 객체를 생성하지 않고 `MemberRequestDto`에서 엔티티로 만들어주면 어떨까요?